### DISCUSSION TITLE
Using OpenSLO Specification with Google SLO Generator

### DISCUSSION THREAD

**Discussion by richiez2** [OpenSLO/OpenSLO#268] [Q&A] [answered]
The documentation in the [Google SLO Generator](https://github.com/google/slo-generator) uses a different specification for configuring SLOs in the YAML files. Is it possible to use the OpenSLO specification in the SLO Generator project from Google? If so, how can this be done? Thanks in advance!

**Comment by nieomylnieja** [comment, marked as answer]
hey @richiez2, I think the only way is to write a custom converter between the two, I don't recall having anyone from Google contact us in regards to writing their spec or reusing ours

**Reply by nieomylnieja** [reply]
What might be of interest to you is our effort to define an SDK in Go for OpenSLO, once that's done it'd be definitely easier to write such converter
